spec/controllers/webhook_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe WebhookController, type: :controller do
```

Comment:
We usually prefer request specs over controller specs. In this case we just have one endpoint so I wouldn't worry about it though.